app/src/main/kotlin/com/saltedge/authenticator/features/onboarding/OnboardingSetupActivity.kt

```diff
@@ -77,7 +77,9 @@ class OnboardingSetupActivity : AppCompatActivity(),
 
     override fun onEnteredPasscodeIsValid() {}
 
-    override fun onEnteredPasscodeIsInvalid() {}
+    override fun onEnteredPasscodeIsInvalid() {
+        viewModel.enteredNewPasscode(PasscodeInputView.InputMode.NEW_PASSCODE)
```

Comment:
move business logic to viewModel, container should redirect events from child view and not to add own data